apps/cookbook/src/app/examples/page-example/fixed-title-and-actions/page-fixed-title-and-actions-example.component.ts

```diff
@@ -26,9 +26,14 @@ const config = {
   template: `<kirby-page toolbarTitle="A Fixed Title" defaultBackHref="/">
   <!-- Fixed Page Actions -->
   <kirby-page-actions *kirbyPageActions="{fixed: true}">
-    <button kirby-button (click)="onMoreSelect()">
-      <kirby-icon name="more"></kirby-icon>
-    </button>
+  <kirby-action-sheet 
+  header="Your action sheet header" 
+  subHeader="Your action sheet subheader"
+  [items]="items"
+  cancelButtonText="Custom cancel"
+  itemSelect="onItemSelect($event)">
```

Comment:
Add missing event handler syntax:
```suggestion
  (itemSelect)="onItemSelect($event)">
```